subgraphs/opyn-gamma/schema.graphql

```diff
@@ -0,0 +1,1395 @@
+  dailyDepositPremiumUSD: BigDecimal!
+
+  " All premium paid to deposit in liquidity pools. "
+  cumulativeDepositPremiumUSD: BigDecimal!
+
+  " Daily premium paid to withdraw from liquidity pools. "
+  dailyWithdrawPremiumUSD: BigDecimal!
+
+  " All premium paid to withdraw from liquidity pools. "
+  cumulativeWithdrawPremiumUSD: BigDecimal!
+
+  " Daily premium paid to enter and exit liquidity pool. "
+  dailyTotalLiquidityPremiumUSD: BigDecimal!
+
+  " All premium paid to enter and exit liquidity pool. "
+  cumulativeTotalLiquidityPremiumUSD: BigDecimal!
+
+  " Daily number of puts minted "
+  dailyPutsMintedCount: Int!
+
+  " Total number of puts minted "
+  putsMintedCount: Int!
+
+  " Daily number of calls minted "
+  dailyCallsMintedCount: Int!
+
+  " Total number of calls minted "
+  callsMintedCount: Int!
+
+  " Daily number of contracts minted "
+  dailyContractsMintedCount: Int!
+
+  " Total number of contracts minted "
+  contractsMintedCount: Int!
+
+  " Daily number of contracts taken"
+  dailyContractsTakenCount: Int!
+
+  " Total number of contracts taken "
+  contractsTakenCount: Int!
+
+  " Daily number of contracts expired "
+  dailyContractsExpiredCount: Int!
+
+  " Total number of contracts expired "
+  contractsExpiredCount: Int!
+
+  " Daily number of contracts exercised "
+  dailyContractsExercisedCount: Int!
+
+  " Total number of contracts exercised "
+  contractsExercisedCount: Int!
+
+  " Daily number of contracts exercised+expired "
+  dailyContractsClosedCount: Int!
+
+  " Total number of contracts exercised+expired "
+  contractsClosedCount: Int!
+
+  " Total number of open positions (contracts minted-closed) "
+  openPositionCount: Int!
+
+  " Total number of closed positions "
+  closedPositionCount: Int!
+
+  " Daily trade notional volume occurred when contract created, in USD "
+  dailyVolumeUSD: BigDecimal!
+
+  " Daily trade notional volume for a specific input token, in native amount. The ordering should be the same as the pool's `inputTokens` field. "
+  dailyVolumeByTokenAmount: [BigInt!]!
+
+  " Daily trade notional volumefor a specific input token, in USD. The ordering should be the same as the pool's `inputTokens` field. "
+  dailyVolumeByTokenUSD: [BigDecimal!]!
+
+  " All historical trade notional volume occurred in this pool, in USD "
+  cumulativeVolumeUSD: BigDecimal!
+
+  " Daily Deposit trade notional volume in USD "
+  dailyDepositVolumeUSD: BigDecimal!
+
+  " Daily Deposit trade notional volume for a specific input token, in native amount. The ordering should be the same as the pool's `inputTokens` field. "
+  dailyDepositVolumeByTokenAmount: [BigInt!]!
+
+  " Daily Deposit trade notional volume for a specific input token, in USD. The ordering should be the same as the pool's `inputTokens` field. "
+  dailyDepositVolumeByTokenUSD: [BigDecimal!]!
+
+  " Deposit historical trade notional volume occurred in this pool, in USD "
+  cumulativeDepositVolumeUSD: BigDecimal!
```

Comment:
Can you keep the original naming of these fields? Since there is another options protocol in the works, to keep them consistent.